editor/test/reducer.js

```diff
@@ -742,7 +742,7 @@ describe( 'state', () => {
 				uid: 'kumquat',
 			} );
 
-			expect( state ).toEqual( { start: 'kumquat', end: 'kumquat', focus: {}, isMultiSelecting: false } );
+			expect( state ).toEqual( { start: 'kumquat', end: 'kumquat', focus: {}, isMultiSelecting: false, isSelectionEnabled: true } );
```

Comment:
These lines are quite long and I'd suggest considering to split them on multiple lines.